eslint.config.mjs

```diff
@@ -51,6 +51,11 @@ export default tseslint.config(
         languageOptions: {
             parserOptions: {
                 warnOnUnsupportedTypeScriptVersion: false,
+                tsconfigRootDir: __dirname,
+                projectService: {
+                    defaultProject: path.join(__dirname, "scripts", "tsconfig.json"),

```

Comment:
ts-eslint internally is calling `getParsedCommandLineOfConfigFile`, on say `"./scripts/tsconfig.json"`, but then passes in a host which sets the cwd to `path.dirname("./scripts/tsconfig.json")`, which isn't quite right.

Passing in an absolute path avoids that, but I believe there's a bug here in ts-eslint.

Reply:
(cc @JoshuaKGoldberg)

Reply:
Nice find! I also suspect there's an additional improvement we should make in reporting a more informative complaint.